ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-rollback-stable.yaml

```diff
@@ -99,4 +99,4 @@ zz_generated_metadata:
   branch: release-4.16
   org: openshift
   repo: openshift-tests-private
-  variant: amd64-rollback-ec
```

Comment:
The last comment is not resolved yet, right?

Reply:
@dis016 Do we need to remove `stream: 4-dev-preview` in this file, too ?

Reply:
@liangxia updated.